docs/roadmap/roadmap.rst

```diff
@@ -0,0 +1,82 @@
+:title: Roadmap
+:description: The Deis project roadmap
+
+.. _roadmap:
+
+Deis Roadmap
+============
+The Deis Roadmap is a community document created as part of our open :ref:`planning`.
+Each roadmap item describes a high-level capability or grouping of features we deem
+important to the future of Deis.
+
+Given our rapid :ref:`Release Schedule, <release_schedule>` roadmap items are designed to provide a sense of
+direction over many releases.
+
+Update Service
+--------------
+Deis must support 100% automated, zero-downtime updates of the control plane.
+Like CoreOS, Deis clusters should be attached to an alpha, beta or stable channel and rely on an automatic update mechanism.
+To accomplish this, we plan to use the `Google Omaha Protocol`_ as implemented by `CoreUpdate`_.
+
+ - [ ] Update client/agent
+ - [ ] Update server
+ - [ ] CI Integration
+
+Scheduling and Orchestration
+----------------------------
+Today Deis uses `Fleet`_ for scheduling.  Unfortunately, Fleet does not support
+resource-based scheduling which results in poor cluster utilization at scale.
```

Comment:
Add a comma before "which," otherwise this might be read as "resource based-scheduling results in poor cluster utilization."